compiler/passes/checkResolved.cpp

```diff
@@ -96,13 +96,26 @@ checkResolved() {
 
   forv_Vec(TypeSymbol, type, gTypeSymbols) {
     if (EnumType* et = toEnumType(type->type)) {
+      std::set<std::string> enumVals;
       for_enums(def, et) {
         if (def->init) {
           SymExpr* sym = toSymExpr(def->init);
           if (!sym || (!sym->symbol()->hasFlag(FLAG_PARAM) &&
-                       !toVarSymbol(sym->symbol())->immediate))
+                       !toVarSymbol(sym->symbol())->immediate)) {
             USR_FATAL_CONT(def, "enumerator '%s' is not an integer param value",
                            def->sym->name);
+          } else if (fWarnUnstable) {
+            Immediate* imm = toVarSymbol(sym->symbol())->immediate;
+            std::string enumval = imm->to_string();
```

Comment:
camelCase enumval maybe?

Reply:
You're not familiar with the English word "enumval"?  :D